app/controllers/api/v1/devlogs_controller.rb

```diff
@@ -41,19 +41,12 @@ class Api::V1::DevlogsController < Api::BaseController
   }
 
   def index
-    @pagy, @devlogs = pagy(
-      Post::Devlog
-        .joins("INNER JOIN posts ON posts.postable_id::bigint = post_devlogs.id AND posts.postable_type = 'Post::Devlog'")
-        .order("post_devlogs.created_at DESC"),
-      limit: 100
-    )
+    devlogs = Post::Devlog.joins(:post).order(created_at: :desc)
```

Comment:
Devlogs do not have a belongs_to: post relationship so using an implicit join breaks it. Please revert this change for both index and show.